pennylane/qchem/openfermion_obs.py

```diff
@@ -812,23 +812,17 @@ def decompose(hf_file, mapping="jordan_wigner", core=None, active=None):
 
 
 def molecular_hamiltonian(
```

Comment:
One should be able to call this function in both ways:

1. Old UI: molecular_hamiltonian(symbols, coordinates, name="molecule", ...)
2. New UI: molecular_hamiltonian(molecule, ...)

Please follow option 1 in this [document](https://app.shortcut.com/xanaduai/write/IkRvYyI6I3V1aWQgIjY2MTk2YTAxLTY1ZmYtNGZlNy04ODUxLTBkNDE1YTdhODM3NiI=).